tests/sentry/integrations/api/endpoints/test_data_forwarding.py

```diff
@@ -0,0 +1,332 @@
+from sentry.integrations.models.data_forwarder import DataForwarder
+from sentry.integrations.models.data_forwarder_project import DataForwarderProject
+from sentry.integrations.types import DataForwarderProviderSlug
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test
+class DataForwardingIndexEndpointTest(APITestCase):
+    endpoint = "sentry-api-0-organization-forwarding"
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.login_as(user=self.user)
+
+
+@region_silo_test
+class DataForwardingIndexGetTest(DataForwardingIndexEndpointTest):
+    def test_get_single_data_forwarder(self) -> None:
+        data_forwarder = DataForwarder.objects.create(
+            organization=self.organization,
+            provider=DataForwarderProviderSlug.SEGMENT,
+            config={"write_key": "test_key"},
+            is_enabled=True,
+        )
```

Comment:
nit: Calling the DB directly works, but we ever want to create these objects in a `control_silo_test` this would produce an error. For that reason we prefer factory methods:

- Add a factory here, linking a simple example, though you can set other defaults to make it easier if you like: https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/factories.py#L1192-L1195
- Add to the fixture (part of the base testcase class) here. https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/fixtures.py#L356-L357

After that you can do `self.create_data_forwarder(**kwargs_that_override_defaults)` in any test you like, region or control